Apps/Database/src/Delegates/DBCommunicationDelegate.cs

```diff
@@ -132,5 +133,15 @@ public DBResult<Communication> Update(Communication communication, bool commit =
             this.logger.LogDebug($"Finished updating Communication in DB");
             return result;
         }
+
+        /// <inheritdoc />
+        public List<Communication> GetEmailCommunicationsInNewProcessingOrError()
```

Comment:
should we not get only the communications for today or past its scheduled date?

Reply:
Do you see any scenarios this can cause issue if we return only the communications based on today and scheduled date? Also for info, schedule date has not been introduced yet.

Reply:
Since this is retrieving communications to send I would just name it that. Something like "GetEmailCommunicationsToSend"